src/Network/Pusher/Data.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveGeneric, OverloadedStrings, MultiWayIf #-}
```

Comment:
I don't think `MultiWayIf` is used?

In general I think we should avoid syntax extensions. It's just more of a barrier to entry.